src/app/App.js

```diff
@@ -1,5 +1,6 @@
 import createWhiteButtonWithIcon from './components/ButtonWhiteWithIcon';
 
+
```

Comment:
It looks like new lines in this file are accidental. Could you remove or revert changes to this file?